### PR TITLE
[panw] Update `event.created` field to follow ECS spec

### PR DESCRIPTION
**Pull request by mjwolf**
## Proposed commit message

The [ECS spec states](https://www.elastic.co/guide/en/ecs/current/ecs-event.html#field-event-created) `event.created` should be the time the event is first seen by the agent or ingested. The panw integration was not following this, and was instead setting it to the PANOS event timestamp.

This corrects the integration to follow ECS properly.

These field changes have been made:
* `event.created` is set from filebeat's initial timestamp (before modification by the syslog processor or ingest pipeline).
*  `panw.panos.received_time` is now defined as the PANOS log timestamp (it is the same value that was previous in `event.created`)
* `panw.panos.generated_time` is added to hold the PANOS generated time
* `@timestamp` now holds the `panw.panos.high_resolution_timestamp` value, or if it isn't available `panw.panos.received_time`

This is also a major version upgrade, because of these changed field definitions.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -26,10 +26,13 @@ processors:
   - rename:
       field: message
       target_field: _temp_.message
+  - set:
+      field: event.created
+      copy_from: "@timestamp"
```
Comment (taylor-swanson) [inline packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/default.yml:31]: Filebeat will set event.created for us. We don't want this to be the value of `@timestamp`:

From the ECS event.created docs:

>This field is distinct from @timestamp in that @timestamp typically contain the time extracted from the original event. In most situations, these two timestamps will be slightly different. The difference can be used to calculate the delay between your source generating an event, and the time when your agent first processed it. This can be used to monitor your agent’s or pipeline’s ability to keep up with your event source.

Comment (mjwolf) [reply packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/default.yml]: Thanks, I didn't realize that.

I've made two changes:
1. removed `event.created` from the pipeline completely.
2. Set `@timestamp` from `panw.panos.received_time` if `panw.panos.high_resolution_timestamp` is not available (the pipeline was already setting it to the high res timestamp, if it existed in the log).

Comment (taylor-swanson) [reply packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/default.yml]: I was completely wrong about `event.created` being set automatically. I was thinking of `event.ingested`, which is set by the ingest node automatically.

Turns out we are still responsible for `event.created` ourselves. In most inputs, `@timestamp` is set by filebeat using the current time, so this is the best representation of `event.created`. The panw integration is a bit more complicated in that it uses the syslog processor, which will extract the syslog header timestamp into `@timestamp`. I still think it's close enough to the `event.created` time to still be useful, but something to keep in mind. I suppose if we were really concerned, we could try adding a `copy_fields` processor to the agent config to copy `@timestamp` to `event.created` BEFORE the syslog processor ran (and that also assumes that `@timestamp` is even accessible at that point).

So to summarize, I would revert item 1 back to what you had. Item 2 should still be fine.

Comment (mjwolf) [reply packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/default.yml]: I've now added back `event.created` to be set from the syslog timestamp

**Review by taylor-swanson — APPROVED** [review]
